samples/apps/bookinfo/route-rule-all-v1.yaml

```diff
@@ -1,13 +1,4 @@
 type: route-rule
-name: productpage-default
```

Comment:
The only service with multiple versions is the reviews service. All the others have just 1 version.